### PR TITLE
config-tools: create json schema path

### PR DESCRIPTION
**Pull request by YuanyuanZhao609**
Create a temporary directroy to write JSON Schema while build packages.

Tracked-On: #7459
Signed-off-by: Yuanyuan Zhao <yuanyuan.zhao@linux.intel.com>

### CODE REVIEW

**Review by NanlinXie — APPROVED** [review]

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: misc/config_tools/scenario_config/jsonschema/converter.py [hunk 1]
```diff
@@ -389,7 +389,11 @@ def main():
         config_tools = config_tools.parent
 
     schema_file = config_tools / 'schema' / 'sliced.xsd'
-    json_schema_file = config_tools / 'configurator' / 'packages' / 'configurator' / 'build' / 'assets' / 'scenario.json'
+
+    json_schema_path = config_tools / 'configurator' / 'packages' / 'configurator' / 'build' / 'assets'
+    if not os.path.exists(json_schema_path):
+        os.makedirs(json_schema_path)
+    json_schema_file = json_schema_path / 'scenario.json'
```
Comment (junjiemao1) [inline misc/config_tools/scenario_config/jsonschema/converter.py:396]: You can replace the above four lines with a single line below:

`json_schema.parent.mkdir(parents=True, exists_ok=True)`

File: misc/config_tools/scenario_config/jsonschema/converter.py [hunk 2]
```diff
@@ -401,9 +405,11 @@ def main():
     json_schema = json.dumps(json_schema, indent='\t')
 
     # Write file and print successful message
-    open(json_schema_file, 'w', encoding='utf-8').write(json_schema)
+    with open(json_schema_file, 'w', encoding='utf-8') as f:
+        f.write(json_schema)
     print("File %s Convert Success. JSON Schema Write To: %s" % (repr(schema_file), repr(json_schema_file)))
 
 
+
```
Comment (junjiemao1) [inline misc/config_tools/scenario_config/jsonschema/converter.py:413]: Why do we need this newline?

Comment (YuanyuanZhao609) [reply misc/config_tools/scenario_config/jsonschema/converter.py]: removed

File: misc/config_tools/scenario_config/jsonschema/converter.py (hunk 1 above)
Comment (YuanyuanZhao609) [reply misc/config_tools/scenario_config/jsonschema/converter.py]: Updated.

**Review by junjiemao1 — APPROVED** [review]
Fine with me. Thanks!

**Pull request merged**